tool-base/src/main/java/io/spine/tools/java/fs/DefaultJavaPaths.java

```diff
@@ -114,6 +122,14 @@ private GeneratedJava(AbstractDirectory projectDir) {
         public SourceDir dir(SourceSetName ssn) {
             return subDir(ssn, ROOT_NAME);
         }
+
+        /**
+         * Obtains the directory for code generated for the source set with the given name by
+         * the custom Spine code generator.
+         */
+        public SourceDir spine(SourceSetName ssn) {
```

Comment:
As discussed vocally, this method and the one related to `gRPC` should probably go to `ProjectExts`.